### PR TITLE
add details about catching all exceptions

### PR DESCRIPTION
**Pull request by stevestar888**

### CODE REVIEW

File: exceptions.rst [hunk 1]
```diff
@@ -63,7 +63,15 @@ trapping ALL exceptions:
         raise e
 
 This can be helpful when you have no idea about the exceptions which may
```
Comment (yasoob) [inline exceptions.rst:65]: can you update "which" to "that" while you are at it? :)

File: exceptions.rst [hunk 2]
```diff
@@ -63,7 +63,15 @@ trapping ALL exceptions:
...
-be thrown by your program.
+be thrown by your program. If you just looking to catch all execptions,
```
Comment (yasoob) [inline exceptions.rst:66]: "If you just" -> "If you are just"

File: exceptions.rst [hunk 3]
```diff
@@ -63,7 +63,15 @@ trapping ALL exceptions:
         raise e
 
 This can be helpful when you have no idea about the exceptions which may
-be thrown by your program.
+be thrown by your program. If you just looking to catch all execptions, 
+but don't actually care about what they are, you can even exclude the 
+``Exception as e`` part. 
+
+Note: catching all exceptions may have unintended consequences because catching
```
Comment (yasoob) [inline exceptions.rst:70]: "Note:" -> "Note::" 

This will just make sure that it renders in a nice blue box on the website :)

**Comment by stevestar888** [comment]
Sorry it took a few days, but I updated the files with your requested changes!

**Comment by yasoob** [comment]
Thanks!

**Pull request merged**